doc/ref_impl/cntt-ri/chapters/chapter03.md

```diff
@@ -197,7 +197,7 @@ This sections describes the readiness of NFVI before the certification process c
 | `req.inf.com.01` | **must** provide compute resources for VM instances.  |
 | `req.inf.com.02` | **should** include industry standard hardware management systems at both HW device and platform level |
 | `req.inf.com.03` | **should** support symmetrical CPU multi-processing with shared memory access as well as multi-threading. |
-| `req.inf.com.04` | **must** be able to support multiple CPU SKU options to support Base, Network Intensive<!---, and Compute Intensive---> infrastructure profiles. |
+| `req.inf.com.04` | **must** be able to support multiple CPU types to support Base, Network Intensive<!---, and Compute Intensive---> infrastructure profiles. |
```

Comment:
Please see comment made in PR #2097

Reply:
@pgoyal01, we have added the term to the glossary. Should we still define it here as well?